types/index.d.ts

```diff
@@ -88,6 +89,7 @@ declare namespace fastifyWebsocket {
     errorHandler?: (this: FastifyInstance, error: Error, connection: SocketStream, request: FastifyRequest, reply: FastifyReply) => void;
     options?: WebSocketServerOptions;
     connectionOptions?: DuplexOptions;
+    preClose?: preCloseHookHandler | preCloseAsyncHookHandler;
```

Comment:
can you please add a test for this typing? we use tsd.

Reply:
Done, the typings seem to be correct